src/browser/index.jsx

```diff
@@ -395,6 +401,12 @@ var MattermostView = React.createClass({
       }
     });
 
+    webview.addEventListener('page-title-updated', function(event) {
```

Comment:
There are no events for this. So in my idea, you can add the prop to [`MattermostView`](https://github.com/mattermost/desktop/blob/v1.3.0/src/browser/index.jsx#L187-L188). It can be know by using `thisObj.state.key===index`.